test/frontend/frontend.bundle.js

```diff
@@ -0,0 +1,22 @@
+class FakeFile extends Blob {
+    constructor(name, data, opt) {
+        super(data, opt);
+        this.name = name;
+    }
```

Comment:
This is the line in package.json that you may want to tweak:

```js
"format": "prettier '{frontend/src/,scripts/,server/,test/}*.js' 'public/*.css' --single-quote --write",
```

Instead of "test/", we'll need "test/**/" due to the nested files in the test directory.
Although, I _think_ if I currently run `npm run format` on the codebase it may break linting, so I'll shut up now.

![](https://i.imgur.com/ju7qV1c.gif)